demisto_sdk/tests/integration_tests/find_dependencies_integration_test.py

```diff
@@ -46,6 +46,8 @@ def test_integration_find_dependencies__sanity(mocker, repo):
 
     # Change working dir to repo
     with ChangeCWD(integration.repo_path):
+        import demisto_sdk.commands.common.update_id_set as uis
+        mocker.patch.object(uis, 'cpu_count', return_value=1)
```

Comment:
why?

Reply:
One of the reasons that Circle's froze on 3.7 is the high usage of processing power, so I reduced it.

`pool = Pool(processes=cpu_count() * 2)` is the line that in charge of the multiprocessing initiation, so changing `cpu_count` return value to 1 still gives you multiprocessing but with only 2 processors, and not the maximum amount.

Reply:
comment that next to the mock